qos_controller/serenity_controller.cpp

```diff
@@ -23,18 +22,25 @@ class SerenityControllerProcess :
     public Process<SerenityControllerProcess> {
  public:
   SerenityControllerProcess(
-      const lambda::function<Future<ResourceUsage>()>& _usage)
+      const lambda::function<Future<ResourceUsage>()> _usage)
       : usage(_usage) {}
 
   Future<list<QoSCorrection>> corrections() {
```

Comment:
As @skonefal  said: "We took Google's C++ Linter
https://google-styleguide.googlecode.com/svn/trunk/cpplint/cpplint.py"

It means we stick to the braces at the end of line (: